package.json

```diff
@@ -12,6 +12,9 @@
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/styles": "^4.11.4",
```

Comment:
These dependencies on the old package are normal to have inside the repository? I mean if we migrate to the new version, there is any reason that we need to maintain also the old version?

Reply:
Yes we need to change the import statements in the files, they are still running on version 4

Reply:
Am working on it right now

Reply:
Any updates?

Reply:
What type of update you are looking for?

Reply:
Will this pull get merged?

Reply:
Mh, I think this PR is not completed? it adds material-ui v5 but the project runs on the material-ui v4. What is the reason to merge this PR?

Reply:
well I need to pull every file I think, as all the import statements will change

Reply:
Add the Material-UI v5 dependencies without using it, don't have enough sense right? It is really confusing I think.

The migration issue #15 will talk about a full migration to material-ui v5 not only add the dependencies

Reply:
Ok will do it in few days